app/components/wallet/index.js

```diff
@@ -3,3 +3,5 @@ export { default as TxParams } from './TxParams';
 export { default as TxSummary } from './TxSummary';
 export { default as TxConfirmation } from './TxConfirmation';
 export { default as TxSent } from './TxSent';
+export { default as NewVault } from './CreateVaultSteps/NewVault';
```

Comment:
place components in new folder 'vault' under folder 'components' for clearer separation

Reply:
fix